src/features/transfer-ownership/FormSteps/Transaction.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+
+import { Wizard } from '@zero-tech/zui/components/Wizard';
+
+import { StepProps } from './types';
+
+import { StepText } from '../TransferOwnership.constants';
+
+export const Transaction: FC<StepProps> = ({ isTransferring }) => {
```

Comment:
nit: I like to make two separate steps for `Waiting for wallet` and `Processing` - just makes things a little more explicit and means you don't need the `isTransferring` prop. Not a big deal though!

Reply:
I'm on the fence with this one. I preferred to have the ```isTransferring``` prop rather than duplicate a chunk of code of two components. I was toying with the idea of scrapping the ```FormSteps``` components all together and replacing with ```<Wizard.Loading /> ...``` as the nesting the wizard doesn't feel completely necessary. Will leave for now but I'll have this on my mind

Reply:
Sweet, I'm all good with `isTransferring`